index.js

```diff
@@ -11,6 +13,7 @@ module.exports = {
   addData: require('./libs/addData'),
   filterData: require('./libs/filterData'),
   updateOrCreate: require('./libs/updateOrCreate'),
-  request: require('./libs/request'),
+  request: requestFactory, // keep the old alias
```

Comment:
I would just add a deprecation warning for us not to rely on this deprecated feature.

```
const deprecate = function (message) {
  return function () {
    console.warn(message)
    return wrapped.apply(this, arguments)
  }
}

request : deprecate(requestFactory, 'Use requestFactory instead of request. It will be removed in cozy-konnector-libs@4')
```

Reply:
Right, here it is